### PR TITLE
Bug 2088606: Update DNS-over-TLS proposal

### PR DESCRIPTION
**Pull request by brandisher**
* Add log lines in the support section.
* Add main PRs including the k8s admission check.

### CODE REVIEW

File: enhancements/dns/forward-dns-over-tls.md [hunk 1]
```diff
@@ -243,7 +243,13 @@ for TLS configuration.
 Implementing this enhancement requires changes in the following repositories:
 
 openshift/api
+* [Addition of DNS-over-TLS Types](https://github.com/openshift/api/pull/1110)
+
 openshift/cluster-dns-operator
+* [Primary Implementation](https://github.com/openshift/cluster-dns-operator/pull/314)
+
+openshift/kubernetes
+* [Admission Check](https://github.com/openshift/kubernetes/pull/1247)
```
Comment (Miciah) [inline enhancements/dns/forward-dns-over-tls.md:252]: <https://github.com/openshift/kubernetes/pull/1247> merged after the release-4.11 branch was cut, so the change is in <https://github.com/openshift/kubernetes/blob/release-4.12/openshift-kube-apiserver/admission/customresourcevalidation/dns/validate_dns.go> but not in <https://github.com/openshift/kubernetes/blob/release-4.11/openshift-kube-apiserver/admission/customresourcevalidation/dns/validate_dns.go>.  Please make sure this is clear in the enhancement document.

Comment (brandisher) [reply enhancements/dns/forward-dns-over-tls.md]: Fixed!

File: enhancements/dns/forward-dns-over-tls.md [hunk 2]
```diff
@@ -243,7 +243,15 @@ for TLS configuration.
...
 
 openshift/api
+* [Addition of DNS-over-TLS Types](https://github.com/openshift/api/pull/1110)
+
 openshift/cluster-dns-operator
+* [Primary Implementation](https://github.com/openshift/cluster-dns-operator/pull/314)
+
+openshift/kubernetes
+* An [Admission Check](https://github.com/openshift/kubernetes/pull/1247) was added in OpenShift 4.12 to address the 
+  overly loose validation. In OpenShift 4.11 we adressed the loose validation mostly in the `cluster-dns-operator`
```
Comment (Miciah) [inline enhancements/dns/forward-dns-over-tls.md:253]:
```suggestion
  overly loose validation. In OpenShift 4.11 we addressed the loose validation mostly in the `cluster-dns-operator` 
```

Comment (brandisher) [reply enhancements/dns/forward-dns-over-tls.md]: Fixed!

**Pull request merged**